staging/src/k8s.io/api/deepcopy_test.go

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
deepcopy is only one part. Why not roundtrip_test.go? We have that already in the internal type packages.